src/web_interface/components/database_routes.py

```diff
@@ -73,6 +74,33 @@ def browse_database(self, query: str = '{}', only_firmwares=False, inverted=Fals
             search_parameters=search_parameters
         )
 
+    @roles_accepted(*PRIVILEGES['pattern_search'])
+    @AppRoute('/database/browse_binary_search_history', GET)
+    def browse_searches(self):
+        page, per_page, offset = extract_pagination_from_request(request, self._config)
+        all_searches = []
+
+        try:
+            with ConnectTo(FrontEndDbInterface, self._config) as conn:
+                # FIXME Use a proper yara parser
```

Comment:
Do you think it would make sense to use something like [plyara](https://pypi.org/project/plyara/)?

Reply:
Good find. Somehow it does not work for me. It gives some internal error from ply. So I think it would be better to do this somewhen in the future.